build.gradle

```diff
@@ -46,12 +46,12 @@ plugins {
 
 allprojects {
   repositories {
-    artifactoryRelease()
-    artifactorySnapshot()
-    mobileLibs()
+//    artifactoryRelease()
+//    artifactorySnapshot()
+//    jitpack()
     google()
     jcenter()
-    jitpack()
+//    mobileLibs()
```

Comment:
This one can be removed too.

Reply:
Got it. Thanks. Will remove in the upcoming one.